config/lib/express.js

```diff
@@ -141,7 +141,7 @@ module.exports.initModulesConfiguration = function (app, db) {
  */
 module.exports.initHelmetHeaders = function (app) {
   // Use helmet to secure Express headers
-  var SIX_MONTHS = 15778476000;
+  var SIX_MONTHS = 15778476;
```

Comment:
yeah I wasn't sure if to add that comment or not because someone who would change it will probably be aware of helmet